.github/workflows/api-docs-repo.yaml

```diff
@@ -71,10 +71,11 @@ jobs:
           docker exec "$container_id" task controller:gen-crd-docs
 
       # After generating the API docs (above), regenerate our index files as now new CRDs can be linked to their docs
+      # Have to force this to ensure the index files are updated
       - name: Generate Index files
         run: |
           container_id=${{ env.container_id }}
-          docker exec "$container_id" task controller:generate-types
+          docker exec "$container_id" task controller:generate-types --force
```

Comment:
The fact we have to force this indicates that there is a problem in the `generate-types` task definition where it is not reading files that it ought to be.

Do we need to update its:
```
    sources:
      - ./bin/{{.GENERATOR_APP}}
      - azure-arm.yaml
```

section?

Sorry for the slow review on this.